app/assets/javascripts/app/pw-toggle.js

```diff
@@ -4,7 +4,7 @@ function togglePw() {
   const inputs = document.querySelectorAll('input[type="password"]');
 
   if (inputs) {
-    for (let i = 0; i < inputs.length; i++) {
+    for (let i = 0; i < inputs.length; i += 1) {
```

Comment:
There is actually a rule override for this case `"allowForLoopAfterthoughts": true`, but I agree this is a silly rule.